tests/js/client/aql/aql-like-optimization.js

```diff
@@ -68,12 +68,86 @@ function likeOptimizationSuite () {
       });
 
       c.insert(docs);
-      c.ensureIndex({ type: "persistent", fields: ["value1"] });
+      c.ensureIndex({ type: "persistent", fields: ["value1"], name: "per" });
     },
 
     tearDownAll: function () {
       db._drop(cn);
     },
+    
+    testDoNotOptimizeForInvertedIndex: function () {
+      let idx = db[cn].ensureIndex({ type: "inverted", fields: ["name"], name: "inv" });
+
+      const queries = [
+        `FOR doc IN ${cn} OPTIONS { indexHint: 'inv', forceIndexHint: true } FILTER LIKE(doc.name, 'a%') RETURN doc`,
+        `FOR doc IN ${cn} OPTIONS { indexHint: 'inv', forceIndexHint: true } FILTER LIKE(doc.name, 'a') RETURN doc`,
```

Comment:
Can you add same test without forceIndexHint?

Reply:
Done